dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -6,7 +6,7 @@ logstashCollector=Logstash Collector
 logstashCollector.desc=Logstash collector gathers data from various sources and forwards the data to a logstash server using Lumberjack protocol.
 
 source=Source
-source.desc=Specifies a source to be used by the logstash collector.
+source.desc=The list of comma-separated sources used by logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection and audit. Note: To use the audit source, enable the Liberty audit feature.
```

Comment:
Change,
"The list of comma-separated sources used by logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection and audit. Note: To use the audit source, enable the Liberty audit feature."
to,
"Specifies a list of comma-separated sources that are used by the logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection, and audit. To use the audit source, enable the Liberty audit feature."

Reply:
Just updated the description